setup.py

```diff
@@ -50,6 +50,9 @@ def is_installed(pkgname: str) -> bool:
                              'gui/res/*/*',
                              'gui/res/*/*/*']},
 
+    data_files = [('share/applications', ['dev/vidify.desktop']),
```

Comment:
The [linter](https://travis-ci.com/vidify/spotify-music-videos/jobs/285527579#L1366) seems to be complaining about this, can you fix it? There shouldn't be before or after `=`, and the next line needs a bit more indentation.

Reply:
Fixed :)

I also added an if statement to the setup.py, because we only want to install the .desktop files to `/usr/share/applications` if we are running on linux, based on [spyder's setup.py](https://github.com/spyder-ide/spyder/blob/master/setup.py)